### ISSUE TITLE
Check for dependencies in config.mjs

### ISSUE THREAD

**Issue by Davidihl** [upleveled/preflight#417]
### Current behaviour:
```
import dotenv from 'dotenv';
```
is used inside of `config.mjs` but preflight `v2.1.14` declares it an unused dependency.
```bash
UpLeveled Preflight v2.1.14
✔ All changes committed to Git
✔ node_modules/ folder ignored in Git
✔ No extraneous files committed to Git
✔ No secrets committed to Git
✔ Use single package manager
✔ Project folder name matches correct format
⚠ No dependency problems
  ✔ Next.js project has sharp installed
  ✖ No unused dependencies
    › Unused dependencies found:
      * dotenv

      Remove these dependencies by running the following command for each dependency:

      ‎  $ pnpm remove <dependency name here>

  ✖ No dependencies without types
    › Algolia error for `@apollo/experimental-nextjs-app-support`: ObjectID does not exist
✔ GitHub repo has deployed project link under About
✔ ESLint
✔ Stylelint
✔ Prettier
✔ ESLint config is latest version
✔ Stylelint config is latest version
✔ Preflight is latest version
```

### Expected behaviour:
Since dotenv is used, preflight should not throw an error here.

**Comment by karlhorky** [comment]
I guess this is an issue in `depcheck` maybe? Wonder if this one is related:

- https://github.com/depcheck/depcheck/issues/733

**Comment by karlhorky** [comment]
Published in `preflight@2.1.15`
